cmd/kubeops/Dockerfile

```diff
@@ -11,7 +11,14 @@ COPY cmd cmd
 ARG VERSION
 
 ARG GOSEC_VERSION="2.12.0"
+ARG GOLANGCILINT_VERSION="1.46.2"
+
 RUN curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v$GOSEC_VERSION
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANGCILINT_VERSION
+
+# Run golangci-lint to detect issues
+RUN golangci-lint run --timeout=10m ./cmd/kubeops/...
+RUN golangci-lint run --timeout=10m ./pkg/...
```

Comment:
Why investing effort in Kubeops if will be removed (hopefully) soon?

Reply:
Mainly for consistency, but also, if want the github action to be run successfully, we need to fix any issue in the existing codebase. So, given that we would be removing the lining issues from Kubeops, why not add the linter in the building stage, as in the rest of go images?

Reply:
I'm not keen on investing effort in soon-to-be-dead code. Isn't it possible to exclude Kubeops part from any linting action?